_posts/2020/2020-12-09-1466.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: rafaga
+date: 2020-12-09
+rid: 1466
+rafagas:
+
+- keyw: travelfromhome, books
+  desc: 'A literary travel agent recommends book sets matching your destination. It shows locations and traveling routes from the selected book set.'
```

Comment:
This is a general comment.

Only one keyword can be used per link, so you need to remove one from each. You can still use the hashtag notation in the description for the removed, so for example in this case:

```suggestion
- keyw: books
  desc: 'A literary travel agent recommends book sets matching your destination. It shows locations and traveling routes from the selected book set #travelfromhome'
```

Reply:
Changes done!!